### PR TITLE
add spider for Guaira-SP

### PR DESCRIPTION
**Pull request by ilzisilva**
Issue #449

### CODE REVIEW

File: data_collection/gazette/spiders/sp_guaira.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+class SpGuairaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3517406"
+    name = "sp_guaira"
+    allowed_domains = ["guaira.sp.gov.br"]
+    start_urls = ["https://guaira.sp.gov.br/category/diarioficial/"]
+    start_date = datetime.date(2013, 6, 19)
+
+    def extract_edition_number(self, text):
+        edition_number_raw = text.split(" ")[-1]
```
Comment (jaswdr) [inline data_collection/gazette/spiders/sp_guaira.py:17]: One idea to improve this method:
```
return ''.join(re.findall(r"[0-9|.]", text))
```

File: data_collection/gazette/spiders/sp_guaira.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        edition_number = ""
+
+        for char in edition_number_raw:
+            if char.isdigit():
+                edition_number += char
+        return edition_number
+
+    def extract_edition_number_extra(self, text):
+        edition_number = ""
```
Comment (jaswdr) [inline data_collection/gazette/spiders/sp_guaira.py:26]: What should be the difference between this and the `extract_edition_number` method?

**Review by jaswdr — CHANGES_REQUESTED** [review]
Just two comments to improve the number extraction methods, everything else LGTM

**Comment by rennerocha** [comment]
Closed as stale.

**Pull request closed without merging**